src/routers/alpha-router/alpha-router.ts

```diff
@@ -909,7 +919,11 @@ export class AlphaRouter
     const blockNumber = partialRoutingConfig.blockNumber ?? this.getBlockNumberPromise();
 
     const routingConfig: AlphaRouterConfig = _.merge(
-      {},
+      {
+        // These settings could be changed by the partialRoutingConfig
+        useCachedRoutes: true,
```

Comment:
Looks like we will need to explicitly override to true in routing-api as well, if we were to implement this way. I'm thinking if we can do the opposite:

```
dontUseCachedRoutes?: boolean
dontWriteToCachedRoutes?: boolean;

routingConfig.dontUseCachedRoutes && !routingConfig.dontUseCachedRoutes
...
```

On the downside, there's even more cognitive load to this complex system

Reply:
nvm, I interpreted wrong. left-side of the lodash merge is `useCachedRoutes: true` and `writeToCachedRoutes: true` and right-side by default is `useCachedRoutes: undefined` and `writeToCachedRoutes: undefined`, so it takes the defined values.